packaging/rhel/syslog-ng.spec

```diff
@@ -573,6 +573,9 @@ fi
 %{_datadir}/%{name}/tools/
 
 %changelog
+* Tue 01 Oct 2019 Laszlo Budai <laszlo.budai@balabit.com> - 3.24.1-1
```

Comment:
`error: bad date in %changelog: Tue 01 Oct 2019 Laszlo Budai <laszlo.budai@balabit.com> - 3.24.1-1`

I guess the output of the following Python snippet should be correct: `Tue Oct 01 2019`
```python
time.strftime("%a %b %d %Y", time.localtime(time.time()))
```

Reply:
or just `date "+%a %b %d %Y"`